tgui/packages/tgui/interfaces/AdminFax.js

```diff
@@ -69,6 +69,7 @@ export const FaxMainPanel = (props, context) => {
           <Button
             icon="eye"
             disabled={!saved}
+            tooltip="Do not forget to close and open the preview after saving."
```

Comment:
I do not understand

Reply:
I'm wondering why users need to do this at all

Reply:
But also no not really

Reply:
> I'm wondering why users need to do this at all

because, i cant reload paper UI. 
and user need to close paper UI and open it, if he wanna to see result of save.
it just hints.

Reply:
They shouldn't have to, that's bad UX--we need to figure out how to just update the paper UI.